analyticsclient/tests/test_users.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+import httpretty
+
+from analyticsclient.exceptions import NotFoundError
+from analyticsclient.tests import ClientTestCase
+
+
+class UserTests(ClientTestCase):
+    def setUp(self):
+        super(UserTests, self).setUp()
+        self.user_id = 10000
```

Comment:
@bradenmacdonald It would be good to use a string instead of an int here (same reason as above).